Sparks_Steven/Assignments/regex.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+def get_matching_words(regex):
+	words = ["aimlessness", "assassin", "baby", "beekeeper", "belladonna", "cannonball", "crybaby", "denver", "embraceable", "facetious", "flashbulb", "gaslight", "hobgoblin", "iconoclast", "issue", "kebab", "kilo", "laundered", "mattress", "millennia", "natural", "obsessive", "paranoia", "queen", "rabble", "reabsorb", "sacrilegious", "schoolroom", "tabby", "tabloid", "unbearable", "union", "videotape"]
+
+	return [word for word in words if re.search(regex, word)]
+
+print get_matching_words(r".*v.*")
+print''
+print get_matching_words(r".*ss.*")
+print''
+print get_matching_words(r"e$")
+print''
+print get_matching_words(r".*b.b.*")
+print''
+print get_matching_words(r".*b.+b.*")
+print''
+print get_matching_words(r".*b.*b.*")
+print''
+print get_matching_words(r".*a.*e.*i.*o.*u*")
+print''
+print get_matching_words(r"[regular expressions]")
```

Comment:
there are redundancies in here. you only need to mention each character you want to match once